src/app/pivot-grid/pivot-grid.sample.ts

```diff
@@ -41,6 +42,9 @@ export class IgxTotalSaleAggregate {
 })
 export class PivotGridSampleComponent {
     @ViewChild('grid1', { static: true }) public grid1: IgxPivotGridComponent;
+    public comfortable: DisplayDensity = DisplayDensity.comfortable;
```

Comment:
These are not needed for this way of setting density directly to the grid property